unfoldingWord/translationCore
# Post installation Git error

#### This pull request addresses:

This fixes #2961, a post-installation issue where the application would complain if git were not installed and then crash.

#### How to test this pull request:

Install the application, then uninstall git. The application should announce that it requires git and provide you with an option to download it.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3308)
